mlflow/experiments.py

```diff
@@ -108,3 +109,15 @@ def rename_experiment(experiment_id, new_name):
     store = _get_store()
     store.rename_experiment(experiment_id, new_name)
     print("Experiment with id %s has been renamed to '%s'." % (experiment_id, new_name))
+
+
+@commands.command("csv")
+@EXPERIMENT_ID
+@click.option("--filename_csv", type=click.STRING, required=True)
+def generate_csv_with_runs(experiment_id, filename_csv):
+    # type: (str, str) -> None
```

Comment:
This line can probably be removed. Unfortunately MLflow still has to support python 2 implicit type inference.

Reply:
Added a test with None and Nan support. A parameter of to_csv method controls how to export Nan value. I kept the default value that is an empty string.

This kind of comment is how to declare the signature with python2 (https://mypy.readthedocs.io/en/latest/python2.html). I often annotate my functions with the signature because mypy catches lot of errors.
If you didn't already set it, I can create an issue to add mypy checks in travis. WDYT ?

Reply:
Ahh I see. My bad. I'll bring up integrating mypy checks with the team, but it doesn't hurt to add an issue for it. Others on github can upvote it, etc.